classes/ClaytonTestClass3.cls

```diff
@@ -0,0 +1,25 @@
+public class ClaytonTestClass3 {
+
+    public static final String THIS_IS_A_CONSTANT = 'TEST';
+    public static final String thisIsAnotherConstant = 'TEST';
+
+    public void AWrongMethodN_a_me_incredible_wow(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.